src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao = new ActorDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
It's bad practice to initialize with Impl classes. We have a session factory in the main class. Use it for initialization (you can add a constructor with ActorDao input parameter and call it from the main class):
```suggestion
    private final ActorDao actorDao;
    public ActorServiceImpl(ActorDao actorDao) {
        this.actorDao = actorDao;
    }
```